Fix internal tests affected by D77980055

masnesral opened the pull request:
Summary: Looks like the change to how full_like is decomposed means these tests need some tweaks

Differential Revision: D77981005

digantdesai commented on exir/tests/test_memory_planning.py:
```diff
@@ -868,18 +868,19 @@ def forward(self, input, label):
         self.assertEqual(
             ep.executorch_program.execution_plan[0]
-            .values[147]
+            .values[149]
```
Not sure how this will pass w/o https://github.com/pytorch/pytorch/pull/144765. But if it does, then the test may not care about it in the first place :)

digantdesai commented on backends/arm/test/passes/test_decompose_cosine_similarity_pass.py:
```diff
@@ -34,7 +34,7 @@ def test_decompose_cosine_similarity_tosa_BI(module):
         "executorch_exir_dialects_edge__ops_aten_sum_dim_IntList": 3,
         "executorch_exir_dialects_edge__ops_aten_pow_Tensor_Scalar": 2,
-        "executorch_exir_dialects_edge__ops_aten_full_like_default": 1,
+        "executorch_exir_dialects_edge__ops_aten_full_default": 1,
```
same here.

digantdesai reviewed (approved):
Stamping to unblock.

masnesral committed:
Fix internal tests affected by D77980055 (#12554)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/12554

Looks like the change to how full_like is decomposed means these tests need some tweaks

Reviewed By: digantdesai

Differential Revision: D77981005
